stash_engine/spec/unit/models/tenant_spec.rb

```diff
@@ -91,7 +91,7 @@ def expect_exemplia(tenant) # rubocop:disable Metrics/AbcSize
         tenant = Tenant.by_domain('example.edu')
         login_path = tenant.omniauth_login_path
         # TODO: don't hard-code this
-        expect(login_path).to eq('https://stash-dev.example.edu/Shibboleth.sso/Login?target=https%3A%2F%2Fstash-dev.example.edu%2Fstash%2Fauth%2Fshibboleth%2Fcallback&entityID=urn%3Amace%3Aincommon%3Aexample.edu&tenant_id=exemplia')
+        expect(login_path).to eq('https://stash-dev.example.edu/Shibboleth.sso/Login?target=https%3A%2F%2Fstash-dev.example.edu%2Fstash%2Fauth%2Fshibboleth%2Fcallback&entityID=urn%3Amace%3Aincommon%3Aexample.edu')
```

Comment:
Yes, not a good idea to hardcode. Its just a test though so alright I guess to address later in a separate PR